WcaOnRails/app/views/competitions/_nearby_competitions_table.html.erb

```diff
@@ -27,6 +29,8 @@
       <td class="text-nowrap text-right">
         <%= link_to_google_maps_dir "#{@competition.kilometers_to(c).round(2)} km", c.latitude_degrees, c.longitude_degrees, @competition.latitude_degrees, @competition.longitude_degrees %>
       </td>
+      <td><%= c.competitor_limit_enabled ? c.competitor_limit : "---" %></td>
+      <td style="text-align: center;"><%= c.competitors.count if c.is_probably_over? %></td>
```

Comment:
Nit: I think we tend to use `class="text-center"` for stuff like this. Slightly shorter, and avoids using inline styles, which can be controversial.

Reply:
I'll use that one, thanks.